self_enrollment_tool/views.py

```diff
@@ -56,7 +57,7 @@ def index(request):
     course_instance_ids = {crs_id[0] for crs_id in course_info.values_list('course_instance_id')}
 
     self_enroll_course_list = SelfEnrollmentCourse.objects.filter(
-        course_instance_id__in=course_instance_ids)
+        course_instance_id__in=course_instance_ids).order_by('-last_updated')
```

Comment:
This seems right. But when I test this on qa, the new records are showing up at the bottom of the table rather than the top?

Reply:
This works, I'll redeploy so we can try again when QA is working